docs/changes/utils/16807.api.rst

```diff
@@ -0,0 +1 @@
+``format_exception()`` is deprecated.
```

Comment:
Probably want to explain why and if there is an alternative, if applicable. Thanks.

Reply:
I can't think of any situations where I would want to use this function, and a quick search on GitHub confirms that nobody else has found a use for this function in any of the Astropy repositories either. It is therefore clear that there is no reason to keep it and there is no reason to bother with recommending alternatives, but I'm not sure I should write a change log entry saying "``format_exception()`` is deprecated because it is completely useless.". My best guess is this function might have served a purpose in Python 2, but I don't know what exception tracebacks looked like back then and learning it just to confirm my guess for the sake of a change log entry is not worth it.

Reply:
I agree with both. Maybe just "... is deprecated, as it is no longer used in astropy and offers at best marginal benefit over regular exception traces."

Reply:
I expanded the change log entry, but with a somewhat different wording than what was suggested. Saying that the function is not used by `astropy` describes `astropy` implementation in too much detail for a change log entry.